README.md

```diff
@@ -1 +1,87 @@
 # VTEX Pages editor
+
+The Pages editor it's platform to dynamically edit a VTEX Store, making possible to select editable components and change its configurations adding or removing content in a simple and straightforward away.
+
+## How to make your Component editable
+
+The editor supports two ways of defining an editable component, thought a static schema structure or a dynamic function, that receives data and create the schema to be displayed.
+
+### Static Schema
+
+Add to your component an `schema` constant, a JSON object that with the following structure:
+
+```
+const schema = {
```

Comment:
Its better put: `const Obj.schema =`

Reply:
Why? That structure its already used to define the schema in the components.